src/modules/dashboard/sagas/actions/stakeMotion.ts

```diff
@@ -0,0 +1,116 @@
+import { call, fork, put, takeEvery } from 'redux-saga/effects';
+import { ClientType, VotingReputationClient } from '@colony/colony-js';
+
+import { Action, ActionTypes, AllActions } from '~redux/index';
+import { TEMP_getContext, ContextModule } from '~context/index';
+import { putError, takeFrom } from '~utils/saga/effects';
+
+import {
+  createTransaction,
+  createTransactionChannels,
+  getTxChannel,
+} from '../../../core/sagas';
+import { transactionReady } from '../../../core/actionCreators';
+
+function* stakeMotionAction({
```

Comment:
This is not an action, it should not be named `stakeMotionAction`, and it should not live in the `actions` folder